### PR TITLE
phpipam cli tool / api tool

### PR DESCRIPTION
**Pull request by Kyle-Fozo**
uoft-phpipam created, uoft-snipeit updated.

-phpIPAM API and CLI tool created.
-Minor addition of location_id and asset_id lookup to snipeIT.
-snipeIT updated to support ansible workflow in phpIPAM.

### CODE REVIEW

**Comment by Kyle-Fozo** [comment]
@alextremblay over to you, sir.

**Review by alextremblay — CHANGES_REQUESTED** [review]

**Comment by Kyle-Fozo** [comment]
This has been done as requested, thanks.

File: projects/phpipam/README.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+    pass edit example-password
+    ```
+    ```console
+    username = "your-name"
+    password = "secure-password"
+    ```
+
+3. Environment variables. Environment variables are loaded from the `PHPIPAM_` namespace. For example, the `foo` configuration option can be set by setting the `PHPIPAM_FOO` environment variable.
+
+Configuration Options:
```
[the remark this answers is not in the document]
Comment (Kyle-Fozo) [reply projects/phpipam/README.md]: This has been done as requested, thanks.

**Review by alextremblay — APPROVED** [review]
LGTM

**Pull request merged**